core/src/CMakeLists.txt

```diff
@@ -170,11 +170,14 @@ IF (Kokkos_ENABLE_IMPL_DESUL_ATOMICS AND KOKKOS_ENABLE_OPENMPTARGET)
   target_link_libraries(kokkoscore PUBLIC atomic)
 ENDIF()
 
+# FIXME_TRILINOS Trilinos doesn't allow for Kokkos to use find_dependency so we
+# just append the flags in cmake/kokkos_tpls.cmake instead of linking with the
+# OpenMP target.
```

Comment:
Why is comment above the DESUL code and not the OpenMP code?

Reply:
Moved it.